css/style.css

```diff
@@ -1425,12 +1425,29 @@ body#Home #content > .intro .download .btn.action:hover
 
 .org_list
 {
+    max-width: 1400px;
+    margin: 0 auto;
+    padding: 10px 0;
+
+    display: -webkit-box;
+    display: -ms-flexbox;
     display: flex;
```

Comment:
`flex` is a standard:

https://developer.mozilla.org/en-US/docs/Web/CSS/flex
https://www.w3.org/TR/css-flexbox-1